Whisper.net/WhisperProcessor.cs

```diff
@@ -692,9 +692,29 @@ private void OnNewSegment(IntPtr state)
             var languageId = nativeWhisper.Whisper_Full_Lang_Id_From_State(state);
             var language = Marshal.PtrToStringAnsi(nativeWhisper.Whisper_Lang_Str(languageId));
 
-            if (options.ComputeProbabilities)
+            var tokens = new WhisperToken[numberOfTokens];
+
+            for (var tokenIndex = 0; tokenIndex < numberOfTokens; tokenIndex++)

```

Comment:
Can we apply this operation only if requested by user?
I'm thinking that many users won't need all tokens and will introduce some latency for them to retrieve them everytime. Maybe we can show it only if "WithTokenLevelTimestamps" is enabled, or create a new setting for this.